presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1036,6 +1036,28 @@ Sync Partition Metadata
 Invalidate Directory List Cache
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Clears the directory listing cache held by the Hive connector.
+
+Full cache flush:
+
+.. code-block:: sql
+
+   CALL hive.system.invalidate_directory_list_cache();
+
+Invalidate a schema:
+
+.. code-block:: sql
+
+   CALL hive.system.invalidate_directory_list_cache(schema_name => 'web');
+
+Invalidate a table:
+
+.. code-block:: sql
+
+   CALL hive.system.invalidate_directory_list_cache(
+       schema_name => 'web',
+       table_name  => 'clicks');
+
```

Comment:
Similar suggestion like the other PR.

IMO, this seems to duplicate what’s already documented just below. If we’d like to include examples with sample values, we could simply add brief one-line examples alongside the existing documentation.